CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Master
 
+- :game_die: Add min/max operations.
```

Comment:
Move this to the bottom of the master section (it's chronological within the sections).